values.yaml

```
@@ -67,7 +67,11 @@ st2:
   config: |
     [api]
     allow_origin = '*'
-
+  
+  #Override Definitions can be added here. 
+  # https://docs.stackstorm.com/latest/packs.html#overriding-pack-defaults
+  override:
```

Would be better to retain the original naming, so it's more obvious for the users:
```suggestion
  overrides:
```

---

I agree, I will get this pushed up!

---

This has been updated now!